skimage/exposure/exposure.py

```diff
@@ -266,6 +266,39 @@ def intensity_range(image, range_values='image', clip_negative=False):
     return i_min, i_max
 
 
+def _output_dtype(out_range):
```

Comment:
I feel like we could choose a more general name that covers `input_dtype` as well. Maybe `dtype_or_range`? Might be a little easier to read.

Reply:
@lagru 

1. the function name should relate to its return value (always a dtype), not its input parameter! So in this case I think you would want `_dtype_from_dtype_or_range`, which is quite a mouthful!
2. The input dtype is never in question, because it comes from the input array, which is given. So I don't think it's super useful to generalise this for now.

Reply:
LOL **ignore me**, I just saw that you were referring to the argument. :man_facepalming: I shouldn't comment on PRs before I've finished my morning coffee! :coffee: :sweat_smile: